cfme/tests/containers/test_projects_dashboard.py

```diff
@@ -0,0 +1,96 @@
+import pytest
+
+from cfme.containers.provider import ContainersProvider
+from cfme.containers.container import Container
+from cfme.containers.image import Image
+from cfme.containers.service import Service
+
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.version import current_version
+
+
+pytestmark = [
+    pytest.mark.uncollectif(lambda provider: current_version() < "5.9"),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1),
+    pytest.mark.provider([ContainersProvider], scope='function')
+]
+
+tested_objects = [Container, Image, Service]
+project_name = 'test-project-dashboard'  # Predefine project for this test
+
+
+def get_api_object_counts(appliance, project_name, provider):
+    """ Fetches amount of Containers/Services/Images from the API per selected project name"""
+    out = {
+        Container: 0,
+        Service: 0,
+        Image: 0
+    }
+    if isinstance(provider, ContainersProvider):
```

Comment:
__REQUIRED__
You should be sure that `provider` is instance of `ContainersProvider`. Is there any alternative "`provider`"?
I'd change this entire function to the following:
```python
assert isinstance(provider, ContainersProvider)
return {
    Container: len(provider.mgmt.list_container(project_name=project_name))
    Service: len(provider.mgmt.list_service(project_name=project_name))
    Image: len(get_container_images_amt(provider, project_name))
}
```

Reply:
Change the function, you are right it looks better like that.